scripts/build_frontend.sh

```diff
@@ -0,0 +1,3 @@
+cd frontend
```

Comment:
This will only work if this bash script is called from the root gradio folder. Might want to put a comment on top clarifying that.

Reply:
Same for all of the other scripts. 

Perhaps also good to put a README in the scripts folder saying that all of the scripts should be run from the root folder

Reply:
actually in the contributing.md it was stated. however let me check scripts and readme as well.